<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextBrowser" name="textTitle">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>90</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>90</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;PoELeveling v.0.0.1.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Вы пользуетесь сырой версией приложения. За последними обновлениями следите на сайте &lt;span style=&quot; font-style:italic;&quot;&gt;Poediary.com&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Для связи с разработчиком используйте следующую почту &lt;span style=&quot; font-style:italic;&quot;&gt;merzoitcorp@gmail.com&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="btnResume">
           <property name="minimumSize">
            <size>
             <width>230</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>230</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Resume</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="cbRespec">
           <property name="text">
            <string>Respect points</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbItems">
           <property name="text">
            <string>Items</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbGems">
           <property name="text">
            <string>Gems</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="rbAlira">
           <property name="text">
            <string>Kill Alira</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbOak">
           <property name="text">
            <string>Kill Oak</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbKraityn">
           <property name="text">
            <string>Kill Kraityn</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbAll">
           <property name="text">
            <string>Kill All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btnOptions">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Options</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStart">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
